specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,687 @@
+            "description": "An error response received from the Policy store service",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete_DataPolicy": {
+            "$ref": "./examples/DataPolicy_Delete.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "DataPolicy_Get"
+        ],
+        "description": "Get data policy by name",
+        "summary": "API to fetch a purview data policy. The API will return the policy in draft state identified by policyName.",
+        "operationId": "DataPolicy_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/policyName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicy"
+            },
+            "headers": {
+              "ETag": {
+                "description": "An HTTP entity tag associated with the policy entity",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy service",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get_DataPolicy": {
+            "$ref": "./examples/DataPolicy_Get.json"
+          }
+        }
+      }
+    },
+    "/data-policies/{policyName}/{state}": {
+      "get": {
+        "tags": [
+          "DataPolicy_Get"
+        ],
+        "description": "Get data policy by name",
+        "summary": "API to fetch a purview data policy. The API will return the policy based on policyName and state",
+        "operationId": "DataPolicy_GetByPolicyNameAndState",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/policyName"
+          },
+          {
+            "$ref": "#/parameters/state"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicy"
+            },
```

Comment:
The `state` parameter seems to be the only difference compared to `/data-policies/{policyName}` 

What is the difference of this API, compared to request `/data-policies/{policyName}` then filter by `state`?

Reply:
Thanks for the feedback. We had two review meetings with the review team. It follows the similar pattern to the key vaults APIs(not exactly same). Below are the final set of APIs after the review. 
![image](https://github.com/Azure/azure-rest-api-specs/assets/23291975/0ddc1492-73fa-4946-adbc-7c3c20d3b526)

Reply:
OK. I was in the meeting. So if Mike approves it, I am good.